source/textInfos/offsets.py

```diff
@@ -457,6 +461,14 @@ def collapse(self,end=False):
 	def expand(self,unit):
 		self._startOffset,self._endOffset=self._getUnitOffsets(unit,self._startOffset)
 
+	def isOutOfBounds(self):

```

Comment:
Please raise NotImplementedError in displayModel.DisplayModelTextinfo, so switching bounded review will always report that it is not supported to do this. This is because the display model only contains text that is on screen.